pyproject.toml

```diff
@@ -31,4 +31,4 @@ build-backend = "setuptools.build_meta"
 optional-tests = [
   "no_python2: run when `python2` extra NOT installed",
   "no_blackd: run when `d` extra NOT installed",
-]
\ No newline at end of file
+]
```

Comment:
I take it tailing \n is part of standard?

Reply:
honestly I don't recall how this change slipped in here (whether it was manual or by some tool), but I can't imagine it being harmful so I'll leaving it alone :p

Reply:
It'll add noise to the diff next time someone adds lines at the end of this file.

It might also be annoying because a lot of tools out there (e.g.: vim) automatically add a trailing newline, so I can imagine people having issues with that (e.g.: since `neovim` automatically adds this, I have to use another tool to actually delete that).

Reply:
Now that #2330 is merged, this should no longer be an issue (or at least, it won't be in the future :)). Looks like there are some conflicts that need to be resolved now due to EOF normalization though.